vicodyn/include/cocaine/vicodyn/balancer/utils.hpp

```diff
@@ -0,0 +1,31 @@
+#include <random>
+#include <vector>
+
+namespace cocaine {
+namespace vicodyn {
+namespace balancer {
+
+auto uint64_rand() -> uint64_t {
+    static thread_local std::mt19937 gen;
+    std::uniform_int_distribution<uint64_t> distribution(0, std::numeric_limits<uint64_t>::max());
```

Comment:
It's up to you, but hiding threads dependent static setup inside public available (almost library) function may appear a bit confusing, at least comments with description and proofs of necessity needed.

Why `distribution` recreated on each call?

`b = std::numeric_limits<T>::max()` is defaults.